src/framework/components/scroll-view/component.js

```diff
@@ -496,6 +506,47 @@ Object.assign(pc, function () {
             }
         },
 
+        // re-enable useInput flag on any descendent that was disabled
+        _enableContentInput: function () {
+            while(this._disabledContentEntities.length) {
+                var e = this._disabledContentEntities.pop();
+                if (e.element) {
+                    e.element.useInput = true;
+                }
+            }
+
+            this._disabledContent = false;
+        },
+
+        // disable useInput flag on all descendents of this contentEntity
+        _disableContentInput: function () {
+            var self = this;
+            var _disableInput = function (e) {
+                if (e.element && e.element.useInput) {
+                    self._disabledContentEntities.push(e);
+                    e.element.useInput = false;
+                }
+
+                var children = e.children;
+                var i, l;
+                for (i = 0, l = children.length; i < l; i++) {
+                    _disableInput(children[i]);
+                }
+            }
```

Comment:
[nit] Would be nicer to use `.bind(this)` here rather than adding the `self` reference.